packages/react/src/elements/components/Banner/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from "react";
```

Comment:
I'd like to use a convention where the root component file of each pattern has the pattern's name. (E.g. `Banner/Banner.js`) and each one has an `index.js` file that exports all the subcomponents for that pattern.

e.g.
```
# Banner/index.js
export { default } from './Banner.js';
export { types, AVAILABLE_TYPES } from './types';
```

My reasoning is that it has two benefits. One, the implementation of the component is separate from the declarations of what we export. Two, you can `Command+T` to open the main file for a component by it's name.